gensim/models/phrases.py

```diff
@@ -455,24 +455,26 @@ def __init__(
             #. "default" - :func:`~gensim.models.phrases.original_scorer`.
             #. "npmi" - :func:`~gensim.models.phrases.npmi_scorer`.
         common_terms : set of str, optional
-            Set of "stop words" to include in bigram phrases, without affecting their scoring.
-            Allows detection of expressions like `bank_of_america` or `eye_of_the_beholder`.
+            Set of "stop words" that may be included within a phrase, without affecting its scoring.
```

Comment:
While analogous & overlapping in some ways with traditional 'stop words', these aren't exactly the same thing. (Among other things, verbs and pronouns don't seem wise to include in `ENGLISH_COMMON_TERMS`.) So, to actually call them 'stop words' seems misleading (& perhaps prone to user errors, such as supplying some larger set instead, like say NLTK's 100-plus word set). 

Similarly, it occurs to me that this feature/role is inconsistently referred-to as 'common terms' here but 'common words' elsewhere. Perhaps the 'word'/'term' variance should be settled one way or the other, and maybe even 'common' is the wrong word for this particular function. 

If this were still identical to the behavior of the Elasticsearch feature that once inspired it, or if the optimal values to provide were exactly the most 'common' words, perhaps the current name makes sense. But to the extent the algorithm/use has changed, and the most actually 'common' terms aren't necessarily the tight values to provide, making a distinct name could make sense. 

Perhaps, 'connector' words? Or 'filler' words? Or 'subsidiary' words? (They're only considered for phrases via attachment to non-subsidiary words.)

Reply:
True. I'll rename to `filler_words`, and remove any mentions of "stop words".

Reply:
@gojomo done in 75caa93, I went with `connector_words`.

It's a mouthful, but captures the purpose better than `filler_words` (which to me implies these words carry no meaning, which is not true).